pkg/ccloudv2/flink_gateway.go

```diff
@@ -76,40 +76,46 @@ func (c *FlinkGatewayClient) ListAllStatements(environmentId, orgId, computePool
 		}
 		allStatements = append(allStatements, statementListResponse.GetData()...)
 		nextUrl := statementListResponse.Metadata.GetNext()
-		pageToken, done, err = extractNextPageToken(flinkgatewayv1alpha1.NewNullableString(&nextUrl))
+		pageToken, done, err = extractNextPageToken(flinkgatewayv1beta1.NewNullableString(&nextUrl))
 		if err != nil {
 			return nil, err
 		}
 	}
 	return allStatements, nil
 }
 
-func (c *FlinkGatewayClient) CreateStatement(statement, computePoolId, identityPoolId string, properties map[string]string, environmentId, orgId string) (flinkgatewayv1alpha1.SqlV1alpha1Statement, error) {
+func (c *FlinkGatewayClient) CreateStatement(statement, computePoolId string, properties map[string]string, serviceAccountId, identityPoolId, environmentId, orgId string) (flinkgatewayv1beta1.SqlV1beta1Statement, error) {
 	statementName := uuid.New().String()[:18]
 
-	statementObj := flinkgatewayv1alpha1.SqlV1alpha1Statement{
-		Spec: &flinkgatewayv1alpha1.SqlV1alpha1StatementSpec{
-			StatementName:  &statementName,
-			Statement:      &statement,
-			ComputePoolId:  &computePoolId,
-			IdentityPoolId: &identityPoolId,
-			Properties:     &properties,
+	statementObj := flinkgatewayv1beta1.SqlV1beta1Statement{
```

Comment:
Can we create `statementObj` outside of this function? The long-term goal is for these functions to be auto-generated. They shouldn't contain any extra logic.

Reply:
done